images/echoserver/echoserver.go

```diff
@@ -159,11 +159,21 @@ func echoHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	writeEchoResponseHeaders(w, r.Header)
 	w.Header().Set("Content-Type", "application/json")
 	w.Header().Set("X-Content-Type-Options", "nosniff")
 	w.Write(js)
 }
 
+func writeEchoResponseHeaders(w http.ResponseWriter, headers http.Header) {
```

Comment:
went with the header approach rather than query bc theres more defined spec for specifying a header multiple times

also, prefer setting each instance of `X-Echo-Set-Header` with one response header name/value to minimize string parsing/interpretation of input format

Reply:
reversing decision on the repeated header given https://www.rfc-editor.org/rfc/rfc7230#section-3.2.2